cubano-webdriver-manager/src/main/java/org/concordion/cubano/driver/web/provider/LocalBrowserProvider.java

```diff
@@ -228,10 +232,24 @@ protected Object toObject(String value) {
 
     public void cleanup() {
         if (driverPath != null) {
-            try {
-                Runtime.getRuntime().exec("taskkill /F /IM " + new File(driverPath).getName());
-            } catch (IOException e) {
-                throw new RuntimeException("Unable to close browser driver", e);
+            boolean isDebug = java.lang.management.ManagementFactory.getRuntimeMXBean().getInputArguments().toString().indexOf("-agentlib:jdwp") > 0;
```

Comment:
Under what circumstances is `-agentlib:jdwp` set?
I'm still concerned that this could impact on users who aren't expecting it.

`isCleanupDriver()` looks good, but the `||` condition below overrides it if `-agentlib:jdwp` is set.

Reply:
-agentlib:jdwp is set by both intellij and eclipse when running in debug mode.  I’m a bit on the fence as to whether it’s a good idea to do this or not, so if you’re strongly against it we can remove it.  

When running our simple tests we did not get any drivers hanging around – except via IE when we went out through the proxy the browser and driver refused to close (I’m choosing to ignore this one as no one in there right mind would be doing this with IE).  We could force it to happen by debugging and then terminating the test.  It’s possible that this might happen more often after running a true test suite and Jimmy will keep an eye out for this and have a look at the Jenkins server in a few days to see if there are any drivers hanging around.

My only reason to keep this isDebug code is that if someone is doing a lot of debugging we could potentially clean up some orphaned drivers and save the developer the need to go searching for some obscure configuration setting in Cubano.  That said, while I've manually killed these orphaned drivers from time to time it's never bothered me enough to go searching for a solution but they really seem to bother Ray.